vmm/src/config.rs

```diff
@@ -302,14 +302,34 @@ impl DiskConfig {
 
 #[derive(Clone, Debug, Deserialize, Serialize)]
 pub struct NetConfig {
+    #[serde(default = "default_netconfig_tap")]
```

Comment:
I don't think that's needed. If the field is missing serde will set that one to `None`.

Reply:
I agree, but I did that on purpose, so that the function `default_netconfig_tap()` can be reused from the `parse()` function, this way we make sure CLI and API behave the same.

Reply:
fair enough.